src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
+  } else {
+    return (days * 40) - 50;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return days * 40;
  }
  
  if (days < 7) {
    return (days * 40) - 20;
  }
  
  return (days * 40) - 50;
```
Вынеси все числа в переменные
Не пиши else, если у тебя есть return внутри if